platforms/gke/base/use-cases/inference-ref-arch/terraform/deploy-comfyui.sh

```diff
@@ -18,6 +18,8 @@ set -o nounset
 
 start_timestamp=$(date +%s)
 
+CUSTOM_NODES_SRC_PATH="modules/python/src/custom_nodes/google_genmedia"
```

Comment:
Recommend changing it to ${ACP_REPO_DIR}/modules/python/src/custom_nodes/google_genmedia .